samples/pushDocumentWithMetadata.ts

```diff
@@ -0,0 +1,23 @@
+import {Source, DocumentBuilder} from '@coveo/push-api-client';
+
+async function main() {
+  const source = new Source('my_api_key', 'my_coveo_organization_id');
+
+  const myDocument = new DocumentBuilder(
+    'https://my.document.uri',
+    'My document title'
+  )
+    .withAuthor('bob')
+    .withClickableUri('https://my.document.click.com')
+    .withData('these words will be searchable')
+    .withFileExtension('.html')
+    .withMetadata({
+      tags: ['the_first_tag', 'the_second_tag'],
+      version: 1,
+      somekey: 'some value',
```

Comment:
I would just add a comment saying that the field should be created in the org and mapped in the source. Otherwise, the user might not understand why he can't see the `somekey` field in his document.